pkg/logs/config/config.go

```diff
@@ -241,7 +273,7 @@ func BuildHTTPEndpointsWithConfig(logsConfig *LogsConfigKeys, endpointPrefix str
 
 	additionals := logsConfig.getAdditionalEndpoints()
 	for i := 0; i < len(additionals); i++ {
-		additionals[i].UseSSL = main.UseSSL
+		additionals[i].UseSSL = !defaultNoSSL
```

Comment:
By setting this to `!defaultNoSSL` it is possible for this to be different than `main.UseSSL` in the vector case when this is configured on line `237`. I think it's safer to leave this as `main.UseSSL`.

Reply:
It makes sense. I will keep that in mind when I will add the `vector.*` option to the config template.